app/globals.css

```
@@ -3,6 +3,11 @@
 @import "tailwindcss";
 @config "../tailwind.config.ts";
 
+/* Terminal Design System - Feature Flag: NEXT_PUBLIC_TERMINAL_DESIGN */
+@import "../styles/fonts.css";
```

Biome (2.3.13)

[error] This `@import` is in the wrong position.

Any `@import` rules must precede all other valid at-rules and style rules in a stylesheet (ignoring `@charset` and `@layer`), or else the `@import` rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)